realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -651,6 +651,10 @@ protected void finalize() throws Throwable {
         super.finalize();
     }
 
+    public SharedRealm getSharedRealm() {
```

Comment:
Excellent.  Fixing.  Sadly, with the way our code is organized, package and protected visibility are, essentially, exactly the same.